Test/Test.csproj

```diff
@@ -3,8 +3,11 @@
     <OutputType>Exe</OutputType>
     <TargetFrameworks>net45;netcoreapp3.0</TargetFrameworks>
   </PropertyGroup>
+
+  <Import Project="../build.props" />
+
   <ItemGroup>
-    <PackageReference Include="protobuf-net" Version="3.0.0-alpha.43" />
+    <PackageReference Include="protobuf-net" Version="2.4.0" />
```

Comment:
Don't go back and forth in your patches. If you do a mistake in an earlier patch, fix that patch, don't fix it in a separate patch in the same series.

And that goes for this whole patch too. "Fix build on Unix", but why is it broken? Because of the first patch? Then the first patch should be fixed.